mw-webapp/src/firebase.ts

```diff
@@ -2,15 +2,26 @@ import {initializeApp} from "firebase/app";
 import {getDatabase} from "firebase/database";
 
 const firebaseConfig = {
-  apiKey: "AIzaSyBMIAbcvbUOpSIOQIm7tS7S3pVoDIlLtBg",
-  authDomain: "masters-way.firebaseapp.com",
-  databaseURL: "https://masters-way-default-rtdb.europe-west1.firebasedatabase.app",
-  projectId: "masters-way",
-  storageBucket: "masters-way.appspot.com",
-  messagingSenderId: "772004885666",
-  appId: "1:772004885666:web:08d1c51678039b51c513fb",
+  apiKey: import.meta.env.FIREBASE_API_KEY,
+  authDomain: import.meta.env.FIREBASE_AUTH_DOMAIN,
+  databaseURL: import.meta.env.FIREBASE_DATABASE_URL,
+  projectId: import.meta.env.FIREBASE_PROJECT_ID,
+  storageBucket: import.meta.env.FIREBASE_STORAGE_BUCKET,
+  messagingSenderId: import.meta.env.FIREBASE_MESSAGING_SENDER_ID,
+  appId: import.meta.env.FIREBASE_APP_ID,
+};
+
+const secondaryFirebaseConfig = {
+  apiKey: import.meta.env.AUTH_FIREBASE_API_KEY,
+  authDomain: import.meta.env.AUTH_FIREBASE_AUTH_DOMAIN,
+  projectId: import.meta.env.AUTH_FIREBASE_PROJECT_ID,
+  storageBucket: import.meta.env.AUTH_FIREBASE_STORAGE_BUCKET,
+  messagingSenderId: import.meta.env.AUTH_FIREBASE_MESSAGING_SENDER_ID,
+  appId: import.meta.env.AUTH_FIREBASE_APP_ID,
 };
```

Comment:
do you use differnet envs for auth and for data? If yes - please use the only one)

Reply:
Yes, because there are Ekaterina's envs for db and mine for auth.

Reply:
Ok, we should get rid of one of them as soon as possible